src/ajax.js

```diff
@@ -575,7 +575,7 @@ jQuery.extend( {
 		}
 
 		// We can fire global events as of now if asked to
-		// Don't fire events if jQuery.event is undefined in an AMD-usage scenario (#15118)
+		// Don't fire events if jQuery.event is undefined in an ESM-usage scenario (#15118)
```

Comment:
```suggestion
		// Don't fire events if jQuery.event is undefined in an ESM-usage scenario (trac-15118)
```

Reply:
Good point. I might do a separate PR in the future to update all the references.